.github/workflows/linku-bot.yml

```
@@ -0,0 +1,40 @@
+    inputs:
+      weekly:
+        description: '주간 랭킹 실행 여부'
+        required: false
+        default: 'false'
+
+jobs:
+  run-bot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)